.github/workflows/ci.yml

```diff
@@ -0,0 +1,62 @@
+---
+name: ci
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - 'v*'
+    branches:
+      - main
+    paths-ignore:
+      - '**.md'
+      - '**.asciidoc'
+  pull_request:
+    paths-ignore:
+      - '**.md'
+      - '**.asciidoc'
```

Comment:
Do we need a "reverse"-workflow in case there is a required status check?

Reply:
That's indeed a good point, let me add it